App.js

```diff
@@ -0,0 +1,250 @@
+
+db.on('error', (error) => console.log(error))
+db.once('open', () => console.log("connected to database"))
+
+app.use(express.json())
+
+
+const options ={
+    definition:{
+        openapi:'3.0.0',
+        info:{
+            title:"Hospital management",
+            version:"1.0.0"
+        },
+       
+        servers:[
+            {
+               url: 'http://localhost:3000/'
+            }
+        
+            ],
+        
+        
+        components:{
+            securitySchemes:{
+                bearerAuth:{
+                    type: "http",
+                    scheme: "bearer",
+                    bearerFormat: "JWT",
+                },
+            },
+        },
+        security:[
+            {
+                bearerAuth: [],
+            },
+        ],
+        },  
+        apis:['App.js','./routes/*.js'] //'./routes/*.js'
+    }
+
+
+const swaggerSpec = swaggerjsdocs(options)
+app.use('/api-docs',swaggerUi.serve,swaggerUi.setup(swaggerSpec))
+
+app.use('/patient',PatientRoute);
+app.use('/doctor',DoctorRoute);
+
+var refreshTokens = []
+
+/**
+ * @swagger
+ *  components:
+ *      schemas:
+ *          Login:
+ *             type: object
+ *             properties:
+ *                 Email:
+ *                     type: string
+ *                 Password:
+ *                      type: string
+ *          doctorSearch:
+ *             type: object
+ *             properties:
+ *                 doctorName:
+ *                     type: string
+ *                 doctorDepartment:
+ *                     type: string
+ *                 hospitalId:
+ *                     type: string
+ *                 hospitalName:
+ *                     type: string
+ */
+
+
+/**
+ * @swagger
+ * /:
+ *  get:
+ *      summary: welcome to site
+ *      description: welcome to site
+ *      responses:
+ *          200:
+ *              description: To test get method
+ */
+app.get('/', async (req,res) => {
+    res.send("welcome to site")
+})
+
+
+
+
+/**
+ * @swagger
+ * /hospitalList:
+ *  get:
+ *      summary: The get hospital data from database  
+ *      description: displaying all hospital data from database
+ *      responses:
+ *          200:
+ *              description: successfully  displaying all product data from database
+ *              
+ */ 
+
+app.get('/hospitalList', async (req,res) =>{
+    const hospital = await Hospital.find()
+    res.json(hospital)
+
+})
+
+/**
+ * @swagger
+ * /login:
+ *  post:
+ *      summary: Login for Users  
+ *      description: Login for User
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: '#/components/schemas/Login'
+ *      responses:
+ *          200:
+ *              description: Login Successfully
+ *              
+ */ 
+
+app.post('/login', async (req,res) => {
+    
+    let Doctor = await Doctors.findOne({doctorEmail :req.body.Email})
```

Comment:
Naming conventions should be followed properly. Check [this](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx)

Reply:
Naming conventions followed [PR](https://github.com/Sankalp-Sachapara/HospitalPractice/pull/2/files/893ce87fe4fe8a5be2c037597a3a6cfbdffc56ac#r1045073095)